DEVELOPER_GUIDE.md

```diff
@@ -6,7 +6,8 @@
 	- [Git Clone OpenSearch Repo](#git-clone-opensearch-repo)
 	- [Publish OpenSearch Feature/Extensions branch to Maven local](#publish-opensearch-feature/extensions-branch-to-maven-local)
 	- [Run OpenSearch-SDK](#run-opensearch-sdk)
-	- [Run OpenSearch](#run-opensearch)
+	    - [Create extensions.yml file](#create_extensions_file)
```

Comment:
Why is this indented under Run OpenSearch-SDK?  It's actually associated with OpenSearch so probably should not be indented.  Might want to also add "for OpenSearch" to the instructions so it's clear it's something different than the SDK step just completed.